cmd/limactl/template.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/limatmpl"
+	"github.com/spf13/cobra"
+)
+
+func newTemplateCommand() *cobra.Command {
+	validateCommand := &cobra.Command{
+		Use:     "template FILE.yaml|URL",
```

Comment:
Probably this should be like `limactl template read` (or `print`), so that we can add more subcommands under `limactl template *`

Reply:
`limactl validate FILE.yaml` should be probably moved to `limactl template validate LOCATOR`

Reply:
I've added `print` and `validate` subcommands. `validate` currently invokes the old `validateAction()`. The `limactl template` command is now hidden, so it can still be modified, even if it is included in an earlier release.